src/opts.ts

```diff
@@ -49,6 +51,7 @@ Options:
   --cache-map    The map of actions source paths to container destination paths or mount arguments
   --scratch-dir  Where the action is stores some temporary files for its processing. Default: 'scratch'
   --skip-extraction  Skip the extraction of the cache from the docker container
+  --container-image  The container image to use for injecting and extracting the cache. Default: 'busybox:1'
```

Comment:
What about calling this `--utility-image`?

Reply:
I don't have a strong preference, though I think "image" without the accompanying "container" could be ambiguous.